source/_docs/guides/jira-integration.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Jira Integration
+description: How to connect Atlassian Jira issue tracking to a Drupal or WordPress site on Pantheon.
+tags: [siteintegrations]
+type: guide
+permalink: docs/guides/:basename/
+date: 5/4/2017
+contributors: scottmassey
+---
+
+Atlassian's Jira issue tracking is one of the most common applications used to manage projects for application development teams. It is part of a larger suite of tools which includes Bitbucket, a code repository, and Hipchat, a team messaging and collaboration application. Jira is extremely customizable, through manual configuration or the use of installable plugins. It allows for integration with tools in the Atlassian suite as well as other common development tools.
+
+In this guide, we will connect a Jira instance to a site on Pantheon. When changes are pushed to Pantheon that reference a jira issue ID, the commit message will appear in the Jira issue's activity log.
+
+
+## What You’ll Need
+
+- A Jira Instance.
+
+- A Drupal or Wordpress site on Pantheon.
+
+- Optional: [Terminus](https://pantheon.io/docs/terminus) installed locally.
+
+## Create a Jira Service Account
+
+1. For security and isolation, on the Jira user administration dashboard, create a user with standard permissions.
+
+![Service account creation](/source/docs/assets/images/integrations/jira/service_account.png)
+
+## Create a Private Folder for Account Credentials.
+
+1. From your local terminal, use the following PHP command to create a JSON file, substituting the below values with your Jira url and the service account's Jira username and password.
+```
+  $> php -r "print json_encode(['jira_url'=>'https://myjira.atlassian.net','jira_user'=>'serviceaccount','jira_pass'=>'secret']);" > secrets.json
+```
+ 
+2. In your Pantheon **files** directory, create a directory called private, if it doesn't exist already. This exact naming convention ensures its contents are not web-accessible. FTP this JSON file to the files/private directory. Repeat this process for all of the sites environments. Each environment should look similar to this:
+
+![Secrets folder](/source/docs/assets/images/integrations/jira/secrets.png)
```

Comment:
files/private/secrets.json is correct. n.b. https://github.com/pantheon-systems/terminus-secrets-plugin can be used to manage secrets in this location also.